> @carlescufi I don't see any failing tests, can you help me find them? All I see are canceled operations.

CI does not complain about the code change itself, but rather the commit message.

```
Error: See https://docs.zephyrproject.org/latest/contribute/guidelines.html#commit-guidelines for more details

1: UC2 Body does not contain a 'Signed-off-by:' line
1: UC6 Body has no content, should at least have 1 line.
3: B6 Body message is missing
Error: See https://docs.zephyrproject.org/latest/contribute/guidelines.html#commit-guidelines for more details

e0aaa822367a98c09e6d2647ffce327483f6ab0c: author email (JP Sugarbroad <jpsugar@amazon.com>) needs to match one of the signed-off-by entries.
Error: Process completed with exit code 1.
```

--- a/dts/bindings/sdhc/zephyr,sdhc-spi-slot.yaml
+++ b/dts/bindings/sdhc/zephyr,sdhc-spi-slot.yaml
@@ -3,3 +3,5 @@ description: Generic Zephyr SPI based SDHC controller
 compatible: "zephyr,sdhc-spi-slot"
 
 include: [spi-device.yaml]
+
+bus: sd

